startup_runners/outer_startup_runner.c

```diff
@@ -35,7 +35,7 @@ main(int argc, char **argv)
 {
     int result, num_per_core_data;
 
-    if (argc != 2) {
+    if (argc < 2) {  // krun passs more args, which are ignored here.
```

Comment:
Typo: "passs".

Also the comment doesn't make sense to me: if Krun passes more than the expected number of args, this program will raise an error (rather than ignoring them).